tests/test_utils.py

```diff
@@ -169,3 +176,80 @@ def test_val_none(self):
         assert "my_model" in card_text
         assert 'pipeline("text-generation", model="username/my_hub_model", device="cuda")' in card_text
         assert "My Trainer" in card_text
+
+
+class TestDataCollatorForChatML(unittest.TestCase):
+    def setUp(self):
+        # Initialize the tokenizer
+        self.tokenizer = AutoTokenizer.from_pretrained("codellama/CodeLlama-7b-Instruct-hf")
```

Comment:
For tests, I'd prefer a smaller model. Or is there something specific with this one?

Reply:
We just just load the tokenizer anyway. Not a big deal then.